dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -404,8 +408,149 @@ ANNO_CLASSSOURCE_CLOSE_EXCEPTION=CWWKC0063W: [ {0} ]: Discarding extra close exc
 ANNO_CLASSSOURCE_CLOSE_EXCEPTION.explanation=An exception occurred when trying to close the resource.
 ANNO_CLASSSOURCE_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
 
-# {0} The name of the class
-# {1} The name of the jar the class was in.
+# # {0} The name of the class
+# # {1} The name of the jar the class was in.
 ANNO_TARGETS_CORRUPT_CLASS=CWWKC0064I: It was not possible to process the class {0} in {1} for annotations as it appears to be corrupt.
 ANNO_TARGETS_CORRUPT_CLASS.explanation=A problem occurred when reading a class file to determine whether there were any annotations of interest. The class is likely to be an invalid class file. As a result, annotations from this class will not be recognized.
 ANNO_TARGETS_CORRUPT_CLASS.useraction=The class can be replaced, removed or this message ignored depending on how important the class is to the function of the application.
+
+<<<<<<< HEAD
+#### Added 07-Feb-2018:
+
+# # {0} A hashcode id used to identify the object emitting the message
+# # {1} A hashcode id used to identify the object causing the exception
+# # {2} The error message from the exception
+ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION=CWWKC0065W: [ {0} ]: The open of child class source [{1}] caused an exception. The message is: {2}
+ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.explanation=The open of the child class source caused an exception.
+ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+
+# # {0} A hashcode id used to identify the object emitting the message
+# # {1} A hashcode id used to identify the object causing the exception
+# # {2} The error message from the exception
+ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION=CWWKC0066W: [ {0} ]: The close of child class source [{1}] failed with an exception. The message is {3}
+ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION.explanation=The close of the child class sourcecaused an exception.
+ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+
+# # {0} A hashcode id used to identify the object emitting the message
+# # {1} an entry name
```

Comment:
Start with uppercase "A" instead of "an"?

Comment applies to similar strings that start with "a"